backend/gn_plugin_depobio/conf_schema_toml.py

```diff
@@ -0,0 +1,19 @@
+from marshmallow import Schema, fields
+
+
+class DemarchesSimplifiees(Schema):
+    URL = fields.String()
```

Comment:
```suggestion
    URL = fields.String(load_default="https://www.demarches-simplifiees.fr/api/v2/graphql")
```

Notamment pour cohérence avec fichier "depobio.toml.example".